module/reset/src/mod_reset.c

```diff
@@ -0,0 +1,165 @@
+/*
+ * Arm SCP/MCP Software
+ * Copyright (c) 2015-2019, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Description:
+ *     Resource reset management support.
+ */
+
+#include <stdbool.h>
```

Comment:
I see a lot of redundant headers included here. Would you be willing scrub this list please?

Reply:
ok ;)